README.md

```diff
@@ -4,11 +4,11 @@ This is a minimal example of using the [Warthog](https://github.com/goldcaddy77/
 
 ## Setup
 
-To get things set up, run `yarn bootstrap`.
+To get things set up, start postgres with `yarn postgres:start`, then `yarn bootstrap` in another tab.
```

Comment:
Some folks will have Postgres installed and just want to use that instance.  Can you modify the instructions so that we have both sets available?

Reply:
Sure, done.